src/main/java/com/github/alina/sesion4_challenge/ExerciseFive.java

```diff
@@ -0,0 +1,20 @@
+package com.github.alina.sesion4_challenge;
+
+//Write a Java program to print the numbers from 1 to 10 using a for loop. As a next step, modify your program to print only the even numbers from 1 to 10.
+public class ExerciseFive {
+    public static void main(String[] args) {
+        System.out.println("Printing numbers from 1 to 10:");
+        for (int i = 0; i <= 10; i++) {
+            System.out.println(i + " i");
+        }
+        System.out.println();
+        for (int i = 0; i <= 10; i++) {
+            if (i % 2 == 0) {
+                System.out.println(i + " ");
+            }
+        }
+    }
+
+}
+
+
```

Comment:
Please consider to review each class before PR, and delete empty rows that we don't need.
![image](https://github.com/alinaitschool/itschool/assets/120710343/52f71597-e93e-4ecf-bcbf-61095ec7e2c2)

Reply:
Fixed.